### DISCUSSION TITLE
Minecraft Pi Edition Trailer

### DISCUSSION THREAD

**Discussion by ghost** [MCPI-Revival/MCPIL#44] [Show and tell]
Here is a trailer I made for Minecraft Pi Edition. I hope you like it!
https://youtu.be/YhZ8UCtfOdU

**Comment by NoozAbooz** [comment]
The trailer looks pretty sick!!! The usage of the Mojang Logo *cough cough* is probably infringing on copyright though *cough cough*

**Reply by ghost** [reply]
Yeah actually, I’ll just censor that.

**Reply by NoozAbooz** [reply]
Maybe use the MCPI Revival or MCPIL logo instead? Both are in the assets repo in this GitHub organization

**Reply by kotsom** [reply]
My son will love this.

**Comment by leha-code** [comment]
Wa
